### ISSUE TITLE
Add more documentation

### ISSUE THREAD

**Issue by rastogiji** [polly3d/mockai#17]
One of the main use cases for a tool like this is testing  applications/clients which use OpenAI and ensure no breaking changes have been introduced.

Add more documentation on how to use this as part of CI/CD pipelines and integration testing

**Comment by polly3d** [comment]
Can you help me to write the doc? I don't have any experience using CI/CD like that.

**Comment by rastogiji** [comment]
Yeah definitely. Let's create a Slack channel for this project and collaborate there.
